controllers/book.js

```diff
@@ -76,8 +90,10 @@ exports.updateBookById = (req, res) => {
 
 
 exports.deleteBookById = (req, res) => {
-    const bookId = req.params.id;// find book by id and delete
-    Book.findOneAndDelete({ _id: bookId })
+    const bookId = req.params.id; // find book by id and delete
+    Book.findOneAndDelete({
+            _id: bookId
+        })
         .then(res.send(`Record with id:${bookId} --> deleted from DB!`))
```

Comment:
better to send book title instead of book id

Reply:
changes applied.